validator_client/src/notifier.rs

```diff
@@ -39,25 +39,32 @@ async fn notify<T: SlotClock + 'static, E: EthSpec>(
     duties_service: &DutiesService<T, E>,
     log: &Logger,
 ) {
-    let num_available = duties_service.beacon_nodes.num_available().await;
+    let (candidate_info, num_available, num_synced) =
+        duties_service.beacon_nodes.get_notifier_info().await;
+    let num_total = candidate_info.len();
+    let num_synced_fallback = num_synced.saturating_sub(1);
```

Comment:
Does this assume that the primary node is synced?

In the scenario where I have 3 beacon nodes, and primary node is not synced, then we'd get `num_synced=2`, and `num_synced_fallback` would be 2 in this case right? or does `num_synced_fallback` mean something else?

Reply:
Ah yeah, `num_synced_fallback` is a bit of a misnomer. Basically it just means "How many nodes do we have in backup". In your example, `num_synced_fallback` would actually be 1, since the primary node is down, node 2 becomes the primary and so there is only 1 backup. This value is only checked to be above 0 in order to set the `ETH2_FALLBACK_CONNECTED` metric anyway.